testdata/supportiety/executable.dyml

```diff
@@ -0,0 +1,44 @@
+#!{
+
+    @@name="supportiety-server"
+    executable {
+
+        services {
+            "server.http"
+            "server.manage"
+            "server.persistence"
+            "server.query"
+        }
+
+        generator go {
+
+            package "example.com/golangee/supportiety"
+
+            lib {
+                name "encoding/json"
+                short "json"
```

Comment:
ich vermute, dass wir später sowieso das Thema "named-import" aufgreifen. Dann würden wir die Verkürzung hier nicht einführen.

Reply:
`short` ist hier die Referenz, die in Services genutzt wird, um sie Unabhängig der Implementierung zu haben. Ein Service kann `json` als Library angeben und in der Executable gebe ich dann an, wie das aufgelöst wird. Ein Go-Generator wird das als import zu `encoding/json` übersetzen, der Rust-Generator macht `serde_json` draus.